tokio-io/src/async_read.rs

```diff
@@ -2,8 +2,9 @@ use std::io as std_io;
 use bytes::BufMut;
 use futures::{Async, Poll};
 
-use {framed, split, AsyncWrite};
-use codec::{Decoder, Encoder, Framed};
+// TODO: Things commented out here while we figure out what to do about AsyncRead::framed
```

Comment:
Copied from the [comment](https://github.com/tokio-rs/tokio/issues/353#issuecomment-388640637)

@bryanburgers yes. We should deprecate `framed` on `AsyncRead` as well. The deprecated `AsyncRead::framed` fn can point to the deprecated `Framed` types in `tokio-io`.

Reply:
Done.